cfgov/v1/models/base.py

```diff
@@ -235,6 +237,16 @@ def get_appropriate_descendants(self, inclusive=True):
     def get_appropriate_siblings(self, inclusive=True):
         return CFGOVPage.objects.live().sibling_of(self, inclusive)
 
+    def remove_html_tags(self, text):
+        clean = re.compile('<.*?>')
+        return re.sub(clean, ' ', text)
```

Comment:
This is outside scope of the PR, but a `strip_with_spaces` utility could be useful in other contexts, like building search indexes.